src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -336,65 +338,64 @@ export default {
 		}
 	},
 	apollo: {
-		preFetch(config, client, args) {
+		preFetch(_config, client, args) {
```

Comment:
Whats the thinking behind renaming these function arguments to add "_" ? Renaming `loan` to `loans` in remaining loans above is ok, but renaming this preFetch functions makes this one different than all the other components. Personally when I see the underscore I immediately think lodash.... but maybe thats just me

Reply:
Adding an underscore prefix is a relatively common practice for designating unused parameters. For example, the Vetur VSCode extension has this as part of its linting rules by default, though I know we don't have that as part of our eslint configs. I can revert these underscores if that's not a practice we want to employ here. It's just something I've seen/used for a while, so I just do it by default when I come across an unused parameter.

Reply:
@eddieferrer Thoughts on using underscores or should I remove for code consistency?

Reply:
Removed underscores for consistency.